### ISSUE TITLE
Should VariableForProperty report lint for `transparent` and `inherit`?

### ISSUE THREAD

**Issue by chris-dura** [brigade/scss-lint#426]
```yml
  VariableForProperty:
    enabled: true
    properties:
      - color
      - font
```
```scss
.foo {
   color: inherit;
   font: inherit;
}

.bar {
   color: transparent;
}
```
This reports lint, but I'm not sure it should?
Similar to my reasoning here: https://github.com/brigade/scss-lint/issues/376#issue-59166822

Writing this seems like a smell...
```scss
$inherit: inherit;
$transparent: transparent;

.foo {
   color: $inherit;
   font: $inherit;
}

.bar {
   color: $transparent;
}
```

**Comment by sds** [comment]
Thanks for the suggestion, @chris-dura.

I've added a quick fix to this in 5a8cac9a2121c37b688712fc28f35e7aaa97dd36.

Can you think of other property values for which it might make sense to ignore? We may need to create a dictionary of values to ignore based on the property in question.

**Comment by lencioni** [comment]
> Can you think of other property values for which it might make sense to ignore?

I added `currentColor` to the list: https://github.com/brigade/scss-lint/pull/433
